# Configuration for `pub_rgbd_and_cloud.py`.

ros_topic_namespace: "my_test"
pub_frequency: 0.5 # How many data to publish per second.
is_loop_forever: True # After reading all images from the folder, loop back to the 1st one.
frame_id: "head_camera" # Name of the camera frame.

rgb:
  is_publish: True
  folder: "/home/feiyu/catkin_ws/src/ros_pub_and_sub_rgbd_and_cloud/data/color/"
  ros_topic: "rgb"

depth:
  is_publish: True
  folder: "/home/feiyu/catkin_ws/src/ros_pub_and_sub_rgbd_and_cloud/data/depth/"
  ros_topic: "depth"

camera_info:
  is_publish: True
  file: "/home/feiyu/catkin_ws/src/ros_pub_and_sub_rgbd_and_cloud/config/cam_params_realsense.json"
  ros_topic: "camera_info"

point_cloud:
  is_publish: False
  ros_topic: "point_cloud"
